whittle/search_sub_networks.py

```
@@ -164,14 +226,33 @@ def main(
     train: TrainArgs,
     eval: EvalArgs,
     search: SearchArgs,
+    param_bins: ParamBinArgs | None = None,
+    objective_1: str = "val_loss",
+    objective_2: str = "parameters",
+    log_objective_names: bool = True,
+    save_checkpoints: bool = True,
+    fine_tuned: bool = False,
+    copy_config_files: bool = True,
+    verbose: bool = True,
 ) -> None:
+    assert objective_1 in [
```

maybe have this assert already in setup to fail fast?